src/docs/Components/DataTable/index.mdx

```diff
@@ -82,6 +136,8 @@ The DataTable footer is placed at the bottom of the component and contains the e
 | **Displayed rows indicator**              | Display the page's rows and the total number of rows  |         |
 | [**Pagination**](/Components/Pagination/) | Allow to navigate to the other pages of the datatable |         |
 
+As it is impossible to be 100% sure how many items will be displayed, it is strongly recommended to always display the footer.
```

Comment:
Move this sentence just after the first sentence 😉

Reply:
~~As it is impossible to be 100% sure how many items will be displayed, it is strongly recommended to always display the footer.~~ As it is impossible to know 100% how many elements will be displayed, it is strongly recommended to always display the footer.

Reply:
Elements -> rows (avoid misunderstanding between rows of the table and UI elements)